lib/smart_answer_flows/inherits-someone-dies-without-will.rb

```diff
@@ -111,10 +111,6 @@ def define
         save_input_as :parents
 
         on_condition(variable_matches(:region, 'england-and-wales')) do
-          on_condition(variable_matches(:partner, 'yes')) do
-            next_node_if(:outcome_21, responded_with('yes'))
-            next_node_if(:siblings?, responded_with('no'))
-          end
           on_condition(variable_matches(:partner, 'no')) do
```

Comment:
@tadast: [This commit note](871d09ada36367b41eb7e9461e1ee61f9f870bc2) suggests that it is **still** possible to reach the `:parents?` and `:siblings?` questions if `partner` is set to "no", so I think we do still need to keep the `on_condition` statement which you mention. Or have I misunderstood?